src/setup.c

```diff
@@ -271,61 +271,6 @@ static void setup_adc1(void)
 	adc_calibrate(ADC1);
 }
 
-/**
```

Comment:
I would keep this function to read battery level, removing the old gyroscope configuration. So I created a new issue #210 to update this function with new hardware changes.